pkg/apis/pipeline/v1beta1/taskrun_types.go

```diff
@@ -52,6 +52,9 @@ type TaskRunSpec struct {
 	// Used for cancelling a taskrun (and maybe more later on)
 	// +optional
 	Status TaskRunSpecStatus `json:"status,omitempty"`
+	// Status message for cancellation.
+	// +optional
+	StatusMessage TaskRunSpecStatusMessage `json:"status_msg,omitempty"`
```

Comment:
As mentioned above in the `RunSpec`, this should be `statusMessage`, not `status_msg`.

Reply:
Done